esrally/rallyd.py

```diff
@@ -20,7 +20,7 @@
 import logging
 import argparse
 
-from esrally import actor, version, exceptions, DOC_LINK, BANNER, PROGRAM_NAME, check_python_version, log
+from esrally import actor, version, exceptions, doc_link, BANNER, PROGRAM_NAME, check_python_version, log
```

Comment:
Same comment here, there are _almost_ sorted, so we could really have them sorted.